websites/D/Dizilla/presence.ts

```diff
@@ -123,7 +111,13 @@ presence.on("UpdateData", async () => {
 			delete presenceData.startTimestamp;
 			delete presenceData.endTimestamp;
 		}
-
-		presence.setActivity(presenceData);
-	} else presence.setActivity();
+	} else if (showName?.textContent && episode?.textContent) {
+		presenceData.details =
+			`${showName?.textContent
+				.charAt(0)
+				.toUpperCase()}${showName?.textContent.slice(0)}` || "Bilinmeyen Dizi";
```

Comment:
Unresolved

Reply:
cuz it errored due to this?

Reply:
It should be passed by the if case its in, if that showName is undefined it will never go in the if case. so the questionmark can be removed on like 116 and 118